Description: Lower the required Qt version to 6.4
 Ubuntu 24.04 (Noble) ships Qt 6.4.2. Whatsie's Qt 6 code outside the
 separately patched WebEngine permission API does not require Qt 6.10.
Author: Reaper <JohnGrimmReaper@disroot.org>
Forwarded: not-needed
Last-Update: 2026-08-13

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,9 +30,9 @@ execute_process(

 string(TIMESTAMP BUILD_TIMESTAMP "%Y-%m-%d %H:%M:%S")

-# Set Qt version - Require Qt 6.10 minimum
+# Set Qt version - Require Qt 6.4 minimum
 set(QT_VERSION_MAJOR 6)
-set(QT_VERSION_MINOR 10)
+set(QT_VERSION_MINOR 4)

 # Set CMake policies
 set(CMAKE_AUTOMOC ON)
